modules/ROOT/pages/patterns/reference.adoc

```diff
@@ -0,0 +1,459 @@
+= Syntax and semantics
+
+This section contains reference material for looking up the syntax and semantics of specific elements of Graph Pattern Matching. 
+
+[[reference-node-pattern]]
+== Node pattern
+
+A node pattern is a pattern that matches a single node.
+It can be used on its own in a clause such as `MATCH` or `EXIST`, or form part of a xref:patterns/reference.adoc#reference-path-pattern[path pattern]. 
+
+See also xref:patterns/concepts.adoc#node-patterns[node pattern concepts].
+
+=== Syntax
+
+[source, syntax, role=noheader]
+----
+nodePattern ::= "(" [ nodeVariable ] [ labelExpression ] // <1>
+    [ propertyKeyValueExpression ] [ "WHERE" booleanExpression ] ")" // <2>
+----
+
+<1> See the section on xref:patterns/reference.adoc#reference-label-expression[label expressions] for more information. 
+<2> See the section on xref:patterns/reference.adoc#reference-property-key-value-expression[property key value expressions] for more information. 
```

Comment:
Given that links embedded in code aren't possible, I think just leave it as is. This adds too much noise in order to introduce a link to a header that will just appear on the left hand navigation panel anyway.